shell/common/animator.cc

```diff
@@ -74,13 +76,24 @@ void Animator::SetDimensionChangePending() {
   dimension_change_pending_ = true;
 }
 
-void Animator::EnqueueTraceFlowId(uint64_t trace_flow_id) {
+void Animator::EnqueueTraceData(const PointerDataPacket& packet,
+                                uint64_t trace_flow_id) {
+  std::vector<PointerData> pointer_datas;
+  size_t pointer_data_count = packet.data().size() / sizeof(PointerData);
+  for (size_t i = 0; i < pointer_data_count; i++) {
+    PointerData data;
+    packet.GetPointerData(i, &data);
+    pointer_datas.push_back(data);
+  }
```

Comment:
@chinmaygarde : why do we `FML_DISALLOW_COPY_AND_ASSIGN(PointerDataPacket)`? It just has a `std::vector<uint8_t>` which seems to be easily copiable to me. If we allow copy, then the code can be further simplified: https://gist.github.com/liyuqian/c531746185e2f82ba7cd38077e3a2a77

Reply:
@dreveman : according to @cbracken , it seems that we only `DISALLOW_COPY_AND_ASSIGN` because no one needs it before. It should be fine to allow copy and assign in your case.